include/image.mk

```diff
@@ -40,6 +40,10 @@ KDIR=$(KERNEL_BUILD_DIR)
 KDIR_TMP=$(KDIR)/tmp
 DTS_DIR:=$(LINUX_DIR)/arch/$(LINUX_KARCH)/boot/dts
 
+# If there is no explicit EXTRA_IMAGE_NAME use the unquoted one from the .config
+ifeq ($(EXTRA_IMAGE_NAME),)
```

Comment:
Do you mean ``ifneq`` instead of ``ifeq``?

``ifeq ($(EXTRA_IMAGE_NAME),)`` here will be evaluated as ``true`` when ``EXTRA_IMAGE_NAME`` is undefined. 

With ``ifneq``, the conditional is ``true`` when ``EXTRA_IMAGE_NAME`` is defined with any value.